.gitignore

```diff
@@ -25,3 +25,9 @@ cmake-build-*/
 *.kdev?
 spack-build*
 build/
+
+# Eclipse project files
+/.project
```

Comment:
Is the preceeding `/` correct? Naïvely I'd say that this looks up the file in the root file system, but I'm no expert on `.gitignore`.

Reply:
I think this just ignores `.project` files in the root of the repository.

Reply:
Yes. Leading `/` looks only for matching files located in the same directory as the `.gitignore` file itself. Without the `/`, the line would match the files in sub-directories as well.